Documents/Automation/EncryptRootVolume/Documents/npark-encryptrootvolume.json

```diff
@@ -15,10 +15,24 @@
       "type": "String",
       "description": "(Optional) The ARN of the role that allows Automation to perform the actions on your behalf.",
       "default": ""
+    },
+    "maxWaitForCreateSnapshot": {
```

Comment:
Specify the dimension (seconds), something like *maxWaitInSecondsForCreateSnapshot*

Reply:
Corrected in the latest commit